src/main/java/io/github/thebusybiscuit/sensibletoolbox/listeners/ElevatorListener.java

```diff
@@ -65,4 +67,10 @@ public void onPlayerSneak(PlayerToggleSneakEvent event) {
         }
     }
 
+    private boolean isOnGround(@Nonnull Player player) {
+        Location location = player.getLocation();
+        Block currentBlock = location.getBlock();
+        return currentBlock.getRelative(BlockFace.DOWN) != null && location.getY() == location.getBlockY();
```

Comment:
[Block#getRelative()](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/block/Block.html#getRelative(org.bukkit.block.BlockFace)) is marked as `@NotNull`, so this null check here is redundant and will never be null.

You probably meant to do `Block#isEmpty()` for the block below.

Reply:
Resolved